File: snippets/src/rand-array.java
// A Java Program to generate test cases  
// for array filled with random numbers 
import java.io.*; 
import java.util.Random; 
  
class GeneratingRandomArrays 
{ 
  
    // the number of runs  
    // for the test data generated 
    static int RUN = 5; 
  
    // minium range of random numbers 
    static int lowerBound = 0; 
  
    // maximum range of random numbers 
    static int upperBound = 1000; 
  
    // minium size of reqd array 
    static int minSize = 10; 
  
    // maximum size of reqd array 
    static int maxSize = 20; 
  
    // Driver Code 
    public static void main (String[] args) throws IOException  
    { 
        Random random = new Random(); 
  
        for(int i = 0; i < RUN; i++) 
        { 
            int size = random.nextInt(maxSize - minSize) +  
                                                minSize; 
            int[] array = new int[size]; 
  
            System.out.println(size); 
  
            for(int j = 0; j < size; j++) 
            { 
                int a = random.nextInt(upperBound - lowerBound) +  
                                                    lowerBound; 
                System.out.print(a + " "); 
            } 
            System.out.println(); 
        } 
    } 
} 

File: snippets/src/rand-array.py
# A Python3 Program to generate test cases 
# for array filled with random numbers 
import random 
  
# the number of runs 
# for the test data generated 
RUN = 5; 
  
# minimum range of random numbers 
lowerBound = 0; 
  
# maximum range of random numbers 
upperBound = 1000; 
  
# minium size of reqd array 
minSize = 10; 
  
# maximum size of reqd array 
maxSize = 20; 
  
# Driver Code 
if __name__ == '__main__': 
  
    for i in range(RUN): 
        size = random.randrange(0, maxSize - minSize) + minSize; 
        array = [0]*size; 
  
        print(size); 
  
        for j in range(size): 
            a = random.randrange(0, upperBound - lowerBound) + lowerBound; 
            print(a, end=" "); 
  
        print(); 
